x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/inference/action/InferenceActionRequestTests.java

```diff
@@ -71,6 +72,84 @@ public void testParsing() throws IOException {
         }
     }
 
+    public void testValidation() throws IOException {
```

Comment:
I think this test could be split into several smaller tests `testValidation_TextEmbedding_Null`, `testValidation_TextEmbedding_Empty`, `testValidation_Rerank_Null` etc.